src/components/Filters/FilterDropdownMenu.test.tsx

```diff
@@ -0,0 +1,179 @@
+import { useState } from "react";
+import { click, render } from "src/utils/rtl";
+import { checkboxFilter } from "./CheckboxFilter";
+import { FilterDropdownMenu } from "./FilterDropdownMenu";
+import { multiFilter } from "./MultiFilter";
+import { singleFilter } from "./SingleFilter";
+import { FilterDefs } from "./types";
+import { GroupByConfig } from "./utils";
+
+describe("FilterDropdownMenu", () => {
+  it("shows filters when button is clicked", async () => {
+    type TestFilter = { status?: string };
+    const r = await render(
+      <TestFilterDropdownMenu<TestFilter>
+        filterDefs={{
+          status: singleFilter({
+            options: [
+              { id: "active", name: "Active" },
+              { id: "inactive", name: "Inactive" },
+            ],
+            getOptionValue: (o) => o.id,
+            getOptionLabel: (o) => o.name,
+          }),
+        }}
+      />,
+    );
+
+    // Filters should not be visible initially
+    expect(r.queryByTestId("filterDropdown_status")).not.toBeInTheDocument();
```

Comment:
There is a newer syntax, of `expect(r.query.filterDropdown_status).not.toBeInTheDocument())` that does the `queryByTestId` thing.